mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+9384654
```

Comment:
Don't do this, check build/git/hooks/post-checkout and the readme that sits next to it on how to avoid this